--- chmlib-0.37/Makefile.in.orig	2005-10-27 14:04:34.000000000 +0200
+++ chmlib-0.37/Makefile.in	2005-10-28 00:32:04.866327152 +0200
@@ -33,13 +33,9 @@
 	${LIBTOOL} --mode=link ${CC} -o $@ $^ ${LDFLAGS} -rpath $(libdir)
 
 install: src/libchm.la
-	chmod a+r src/libchm.la
 	mkdir -p $(DESTDIR)$(prefix) $(DESTDIR)$(libdir) $(DESTDIR)$(includedir)
-	install -m0755 src/libchm.la $(DESTDIR)$(libdir)/
+	${LIBTOOL} --mode=install install src/libchm.la $(DESTDIR)$(libdir)/
 	install -m0644 $(srcdir)/src/chm_lib.h $(DESTDIR)$(includedir)/
-	install -m0755 src/.libs/libchm.so.0.0.0 $(DESTDIR)$(libdir)/
-	ln -s $(libdir)/libchm.so.0.0.0 $(DESTDIR)$(libdir)/libchm.so
-	ln -s $(libdir)/libchm.so.0.0.0 $(DESTDIR)$(libdir)/libchm.so.0
 
 clean:
 	rm -fr src/libchm.la src/*.o src/*.lo .libs src/.libs src/${EXAMPLES}
