diff -urN chmlib.orig/Makefile chmlib/Makefile
--- chmlib.orig/Makefile	2004-07-05 15:04:04.000000000 +0200
+++ chmlib/Makefile	2004-07-05 15:02:57.000000000 +0200
@@ -11,6 +11,7 @@
 CFLAGS=-g -O2 -DCHM_MT -DCHM_USE_PREAD -DCHM_USE_IO64 -DDMALLOC_DISABLE 
 LDFLAGS= -lpthread
 INSTALLPREFIX=/usr/local
+LIBDIR=/usr/local/lib
 CC=gcc
 top_builddir=
 ifeq ($(top_builddir),)
@@ -27,28 +28,28 @@
 	${LIBTOOL} --mode=compile ${CC} -c -o $@ $^ ${CFLAGS}
 
 src/libchm.la: src/chm_lib.lo src/lzx.lo
-	${LIBTOOL} --mode=link ${CC} -o $@ $^ ${LDFLAGS} -rpath ${INSTALLPREFIX}/lib
+	${LIBTOOL} --mode=link ${CC} -o $@ $^ ${LDFLAGS} -rpath ${LIBDIR}
 
 install: src/libchm.la
 	chmod a+r src/libchm.la
-	${LIBTOOL} --mode=install cp src/libchm.la ${INSTALLPREFIX}/lib
+	${LIBTOOL} --mode=install cp src/libchm.la ${LIBDIR}
 	cp src/chm_lib.h ${INSTALLPREFIX}/include
 
 clean:
 	rm -fr src/libchm.la src/*.o src/*.lo .libs src/.libs src/${EXAMPLES}
 
 test_chmLib: src/test_chmLib.c
-	${CC} -o $@ $^ -I${INSTALLPREFIX}/include -L${INSTALLPREFIX}/lib -lchm ${CFLAGS}
+	${CC} -o $@ $^ -I${INSTALLPREFIX}/include -L${LIBDIR} -lchm ${CFLAGS}
 
 enum_chmLib: src/enum_chmLib.c
-	${CC} -o $@ $^ -I${INSTALLPREFIX}/include -L${INSTALLPREFIX}/lib -lchm ${CFLAGS}
+	${CC} -o $@ $^ -I${INSTALLPREFIX}/include -L${LIBDIR} -lchm ${CFLAGS}
 
 enumdir_chmLib: src/enumdir_chmLib.c
-	${CC} -o $@ $^ -I${INSTALLPREFIX}/include -L${INSTALLPREFIX}/lib -lchm ${CFLAGS}
+	${CC} -o $@ $^ -I${INSTALLPREFIX}/include -L${LIBDIR} -lchm ${CFLAGS}
 
 extract_chmLib: src/extract_chmLib.c
-	${CC} -o $@ $^ -I${INSTALLPREFIX}/include -L${INSTALLPREFIX}/lib -lchm ${CFLAGS}
+	${CC} -o $@ $^ -I${INSTALLPREFIX}/include -L${LIBDIR} -lchm ${CFLAGS}
 
 chm_http: src/chm_http.c
-	${CC} -o $@ $^ -I${INSTALLPREFIX}/include -L${INSTALLPREFIX}/lib -lchm -lpthread ${CFLAGS}
+	${CC} -o $@ $^ -I${INSTALLPREFIX}/include -L${LIBDIR} -lchm -lpthread ${CFLAGS}
 
