diff -urN chmlib.orig/src/chm_lib.c chmlib/src/chm_lib.c
--- chmlib.orig/src/chm_lib.c	2004-05-03 23:29:04.000000000 +0200
+++ chmlib/src/chm_lib.c	2004-07-05 14:38:31.000000000 +0200
@@ -157,14 +157,15 @@
 /* Sparc        */
 /* MIPS         */
 /* PPC          */
-#elif __i386__ || __sun || __sgi || __ppc__
+#elif __i386__ || __sun || __sgi || __ppc__ || __linux__
+#include <stdint.h>
 typedef unsigned char           UChar;
-typedef short                   Int16;
-typedef unsigned short          UInt16;
-typedef long                    Int32;
-typedef unsigned long           UInt32;
-typedef long long               Int64;
-typedef unsigned long long      UInt64;
+typedef int16_t                 Int16;
+typedef uint16_t                UInt16;
+typedef int32_t                 Int32;
+typedef uint32_t                UInt32;
+typedef int64_t                 Int64;
+typedef uint64_t                UInt64;
 #else
 
 /* yielding an error is preferable to yielding incorrect behavior */
@@ -1123,7 +1124,7 @@
      *      that is provided for us.
      */
     struct chmPmglHeader header;
-    UInt32 hremain;
+    unsigned long hremain;
     UChar *end;
     UChar *cur;
     UChar *temp;
@@ -1165,7 +1166,7 @@
      *      that is provided for us
      */
     struct chmPmgiHeader header;
-    UInt32 hremain;
+    unsigned long hremain;
     int page=-1;
     UChar *end;
     UChar *cur;
